src/styles/sections/_program.scss

```diff
@@ -0,0 +1,43 @@
+.program {
+  position: relative;
+  overflow: hidden;
+  background-image: url(@/assets/backgrounds/program.png);
+
+  .container {
+    width: 100%;
+    height: 100%;
+    min-height: 100vh;
```

Comment:
- [ ] You should consider a different approach to this height, since your items are separated with a `space-between`. In the original design the gap between the programs cards and the button is not that big, as you can see here:

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142066179-a55739f5-7e2b-4923-8c0f-a6b7459f0300.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/142066080-4e9f0997-edc0-4225-aaf2-15313b14cce0.png)

Reply:
Solved